mail_tracking_mailgun/models/mail_tracking_email.py

```diff
@@ -27,43 +46,30 @@ def _country_search(self, country_code):
             return country.id
         return False
 
-    @property
-    def _mailgun_mandatory_fields(self):
-        return ('event', 'timestamp', 'token', 'signature',
-                'tracking_email_id', 'odoo_db')
+    @api.model
+    def _mailgun_event2type(self, event, default="UNKNOWN"):
+        """Return the ``mail.tracking.event`` equivalent event
 
-    @property
-    def _mailgun_event_type_mapping(self):
-        return {
-            # Mailgun event type: tracking event type
+        Args:
+            event: Mailgun event response from API.
+            default: Value to return when not found.
+        """
+        # Mailgun event type: tracking event type
+        equivalents = {
```

Comment:
Shouldn't this be a "constant" on top of the file? At least is only populated once.